### DISCUSSION TITLE
Help with Azure OIDC SSO

### DISCUSSION THREAD

**Discussion by nnellanspdl** [argoproj/argo-cd#10688] [Q&A] [answered]
Trying to get this configured and running into some issues.  Does anybody know what I'm doing wrong?

When I go to my ArgoCD url:  `https://myArgoUrl` I get redirected to `https://myArgoUrl/login?return_url=https%3A%2F%2FmyArgoUrl%2Fapplications` which is fine.

Now, when I click on Log In Via Azure I go to `https://myArgoUrl/auth/login?return_url=https%3A%2F%2FmyArgoUrl%2Fapplications` which shows me this error message: `Invalid redirect URL: the protocol and host (including port) must match and the path must be within allowed URLs if provided`

My ConfigMap for argocd-cm is configured for a url of `https://myArgoUrl`

**Comment by nnellanspdl** [comment, marked as answer]
Disregard.  I figured out we have some automation which is overwriting the value of `url` in the argocd-cm.

**Reply by sushruth-k-mn** [reply]
@nnellanspdl  how you resolved the issue
I am getting the same after I upgraded Argocd to latest version

**Reply by unique-dominik** [reply]
@nnellanspdl Saved me ❤️

**Reply by aomaryoussef** [reply]
@nnellanspdl how to solve this issue?

**Comment by onkar6699** [comment]
Hi @nnellanspdl @sushruth-k-mn I am facing same issue running argocd behind nginx

**Comment by ThomasSteinbach** [comment]
@nnellanspdl Awesome. You made my day. I've never set 'global.hostname' but just the hostname for the ingress.
